completion/ctex.cwl

```diff
@@ -0,0 +1,278 @@
+# ctex package
+# Darcy Hu <hot123tea123@gmail.com> 2016
+#modified zepinglee 26 Jan 2021
+
+#include:zhnumber
```

Comment:
Currently TeXstudio cannot load cwl by current engine. Since `ctex` loads `xeCJK` when xetex is used and `ctex` + xetex is popularly used, perhaps adding `#include:xeCJK` here is useful.

Reply:
The `xeCJk` is not loaded in when compiling with pdflatex or lualatex. I suggest explicit `\usepackage{xeCJK}` if the completion for it is needed.

Reply:
I don't have a strong opinion here. After all, cwl cannot be as accurate as tex.